### PR TITLE
Bugfix for mesh generation

### PR DESCRIPTION
**Pull request by mfs4rd**
minor fixes to mesh generation code. Tested on quilt data

### CODE REVIEW

**Review by vianamp — DISMISSED** [review]
Thanks Filip

**Comment by mfs4rd** [comment]
Heads up that I accidentally included the work on the ome_zarr importer script in here as well

**Comment by mfs4rd** [comment]
@smishra3 @pgarrison can one of you please review this pr so I can merge

File: CollagenIV_mesh_generation/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 pyvista==0.43.3
 trimesh==4.1.7
 open3d==0.18.0
-pyacvd==0.2.10
\ No newline at end of file
+pyacvd==0.2.10
+rtree==1.3.0
```
Comment (pgarrison) [inline CollagenIV_mesh_generation/requirements.txt:5]: MIT license is compatible with the AISL 👍

File: Data_access/OME_zarr_importer.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+        # import pdb; pdb.set_trace()
+
+        dtype = image.dtype
+
+        OmeTiffWriter().save(
+            data=image.data,
+            uri=import_path,
+            # ToDO: Add the following arguments to the save method
+            # ome_xml=OmeTiffWriter().build_ome(data_shapes=image.shape,dimension_order=[d for d in image.dims.order],channel_names=image.channel_names,physical_pixel_sizes=image.physical_pixel_sizes,)
```
Comment (pgarrison) [inline Data_access/OME_zarr_importer.py]: Should these arguments be added? I recommend to either uncomment or delete.

File: Data_access/OME_zarr_importer.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            remote_path: str
+                The path to the OME-Zarr file in the s3 bucket. Must begin with \"https://allencell\".
+        """
+        self.check_path(remote_path)
+        image = BioImage(remote_path, reader=bioio_ome_zarr.Reader)
+
+        fname = Path(remote_path).name.replace(".ome.zarr", ".ome.tiff")
+        import_path = Path(self.local_path) / fname
+
+        # import pdb; pdb.set_trace()
```
Comment (pgarrison) [inline Data_access/OME_zarr_importer.py]: Commented out code for debugging should be removed.

**Review by pgarrison — DISMISSED** [review]
Looks good. One day we'll need to write end-to-end instructions for how to put this all together, but that is coming along in a separate PR.

File: Data_access/OME_zarr_importer.py (hunk 2 above)
Comment (mfs4rd) [reply Data_access/OME_zarr_importer.py]: Added

File: Data_access/OME_zarr_importer.py (hunk 3 above)
Comment (mfs4rd) [reply Data_access/OME_zarr_importer.py]: removed

**Review by pgarrison — APPROVED** [review]

**Review by smishra3 — APPROVED** [review]

**Pull request merged**